uber/templates/accounts/index.html

```diff
@@ -79,9 +74,20 @@ <h3 class="panel-title">Existing Accounts</h3>
             </form>
           </td>
           <td>
-            <!--<input type="hidden" name="id" value="{{account.id}}"/ />-->
             <a href="update_password_of_other?id={{account.id}}">Change Password</a>
           </td>
+          <td class="text-nowrap">
+            <button type="submit" class="btn btn-sm btn-primary update-button">
+              <span class="glyphicon glyphicon-ok"></span>
+            </button>
+            <form class="form delete-form" method="post" action="delete">
+              {{ csrf_token() }}
+              <input type="hidden" name="id" value="{{ account.id }}" />
+              <button type="submit" type="button" class="btn btn-sm btn-danger delete-button" data-last_first="{{ account.attendee.last_first }}">
+                <span class="glyphicon glyphicon-trash"></span>
+              </button>
+            </form>
+          </td>
```

Comment:
Moved the save/delete buttons to the right side of the table to match the way they're set up in the view promo codes table.